packages/vsphere-cpi/charts/templates/vsphere-cpi-ds.yaml

```diff
@@ -0,0 +1,74 @@
+# Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/vsphere-cloud-controller-manager.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  labels:
+    component: cloud-controller-manager
+    tier: control-plane
+    k8s-app: vsphere-cloud-controller-manager
+  name: vsphere-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      k8s-app: vsphere-cloud-controller-manager
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        k8s-app: vsphere-cloud-controller-manager
+    spec:
+      nodeSelector:
+      {{- if .Values.nodeSelector}}
+      {{- with .Values.nodeSelector }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- else }}
+        node-role.kubernetes.io/controlplane: "true"
+      {{- end }}
+      securityContext:
+        runAsUser: 1001
+      tolerations:
+      - effect: NoExecute
+        operator: Exists
+      - effect: NoSchedule
+        operator: Exists
+      serviceAccountName: cloud-controller-manager
+      containers:
+        - name: vsphere-cloud-controller-manager
+          image: gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.2.1
```

Comment:
@PennyScissors which image should be replaced here? Can you also reference the image-mirror PR in this comment?

Reply:
This has to be moved to values.yaml in repository:tag format.

Reply:
Addressed in latest, resolving this comment in favor of a new one that points to the right location that needs to be updated.